c10/util/Half.h

```diff
@@ -428,10 +428,9 @@ struct alignas(4) complex<Half> {
 // The overflow checks may involve float to int conversion which may
 // trigger precision loss warning. Re-enable the warning once the code
 // is fixed. See T58053069.
-#ifdef __clang__
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunknown-warning-option"
-#pragma GCC diagnostic ignored "-Wimplicit-int-float-conversion"
+C10_CLANG_DIAGNOSTIC_PUSH()
```

Comment:
It was "GCC diagnostic push", but now "clang diagnostic push", is it OK?

Reply:
Yes, for `clang` those are aliases, and it was guarded under `#ifdef __clang__`